pkgs/development/compilers/flutter/default.nix

```
@@ -4,18 +4,18 @@ let
   getPatches = dir:
     let files = builtins.attrNames (builtins.readDir dir);
     in map (f: dir + ("/" + f)) files;
-  version = "2.0.3";
+  version = "2.2.1";
   channel = "stable";
   filename = "flutter_linux_${version}-${channel}.tar.xz";
-in
-{
+in {
   mkFlutter = mkFlutter;
   stable = mkFlutter rec {
     inherit dart version;
     pname = "flutter";
     src = fetchurl {
-      url = "https://storage.googleapis.com/flutter_infra/releases/${channel}/linux/${filename}";
-      sha256 = "14a63cpkp78rgymmlrppds69jsrdarg33dr43nb7s61r0xfh9icm";
+      url =
+        "https://storage.googleapis.com/flutter_infra/releases/${channel}/linux/${filename}";
```

```suggestion
      url = "https://storage.googleapis.com/flutter_infra/releases/${channel}/linux/${filename}";
```

I would suggest using nixpkgs-fmt instead of nix-fmt.

---

sure, will fix that. I thought my editor was already using nixpkgs-format actually 😅